CONTRIBUTING.md

```diff
@@ -1,65 +1,41 @@
+First of all, double-check that it does not already exist in the [database](./termsTypes.json). If it does, use the associated type for tracking. 
 
-### Build consensus
+Then check the [open discussions](https://github.com/opentermsarchive/terms-types/discussions) for this repository to make sure that there is not already a discussion about the type you intend to add.
 
-Publicise your suggestion across Open Terms Archive channels and engage the community. Throughout the discussion, the name and the triptych should be challenged, taking into account international perspectives on phrasings and the variety of definitions across jurisdictions.
+If no discussion seems to match the type you are interested in adding, [start a new discussion](https://github.com/OpenTermsArchive/terms-types/discussions/new?category=submitted) by filling in this form.
 
-In order to ensure global relevance, your suggestion will only be considered for inclusion when there are at least three examples of such terms, each from different service providers. You are welcome to provide several examples on your own.
+### Integration process
 
-### Integration
+The Core Team reviews the term type additions categorized as `Submitted` to ensure that all the necessary information is gathered and that at least 5 examples are provided by maximising industry and jurisdiction diversity. When this is the case, the discussion is categorized as `Qualified` and opened up to the community.
 
-Once:
+Build consensus by publicizing your suggestion across Open Terms Archive channels and engaging the community. Throughout the discussion, the data should be critically examined, considering international perspectives on phrasing and the variety of definitions across jurisdictions.
 
-- at least three examples from different service providers have been given;
-- a consensus has been reached on the triptych;
-- a consensus has been reached on the name;
-- at least 2 weeks have elapsed since the opening of the discussion, to ensure visibility throughout the community.
+Once a consensus has been reached and at least 2 weeks have passed since the discussion was opened to ensure visibility within the community, a pull request for the addition to the database must be submitted. To facilitate tracking and participation, each pull request will contain only one type addition and will link to the corresponding discussion.
 
-Then the Open Terms Archive Core Team will review and validate the suggestion, considering factors like uniqueness, clarity, and relevance. If validated, the discussion will be turned into a pull request for addition into the database. To ease tracking and participation, each pull request will contain only one type addition, and will link to the discussion.
+The Core Team will then review the pull request, taking into account factors such as uniqueness, clarity, and relevance.
```

Comment:
I feel that the final review should only be about formatting, and that any concerns on uniqueness, clarity and relevance should have been addressed in the discussion 🤔

Reply:
yes!